src/pages/status-change-event-page/status-change-event-validation.ts

```diff
@@ -1,8 +1,8 @@
 import Joi, { ValidationErrorItem } from "joi";
 import { ValidateFormParams } from "../FormPageHoc";
-import { eventSchema } from "../event-schema";
+import { stageSchema } from "../event-schema";
 
-const statusChangeEventSchema = eventSchema.concat(
+const statusChangeEventSchema = stageSchema.concat(
```

Comment:
I hadnt thought of this but this validation as it is cannot use the stageSchema the way it was using the eventSchema before

Reply:
we will have to add the fields we had in eventSchema in this schema and get rid of the concat bit:
i.e

```
  const statusChangeEventSchema =Joi.object({
    eventDate: eventDateSchema,
    notes: Joi.string().allow("").max(100).messages({
      "string.max": `Notes must be less than or equal to 100 characters long`,
    }),
    developmentPlanEvent: Joi.required().messages({
        "any.required": "Select the status of your Local Plan",
      }),
  })

```

Reply:
note that this status change events don't belong to a stage so they have to be validated differently